src/routes/safe/components/Layout/Header/index.jsx

```diff
@@ -0,0 +1,80 @@
+// @flow
+import { withStyles } from '@material-ui/core/styles'
+import CallMade from '@material-ui/icons/CallMade'
+import CallReceived from '@material-ui/icons/CallReceived'
+import classNames from 'classnames/bind'
+import React from 'react'
+import { useSelector } from 'react-redux'
+
+import { styles } from './style'
+
+import CopyBtn from '~/components/CopyBtn'
+import EtherscanBtn from '~/components/EtherscanBtn'
+import Identicon from '~/components/Identicon'
+import Block from '~/components/layout/Block'
+import Button from '~/components/layout/Button'
+import Heading from '~/components/layout/Heading'
+import Paragraph from '~/components/layout/Paragraph'
+import Row from '~/components/layout/Row'
+import { SAFE_VIEW_NAME_HEADING_TEST_ID } from '~/routes/safe/components/Layout'
+import { grantedSelector } from '~/routes/safe/container/selector'
+import { safeNameSelector, safeParamAddressFromStateSelector } from '~/routes/safe/store/selectors'
+
+type Props = {
+  classes: Object,
+  showSendFunds: Function,
+  onShow: Function,
+}
+
+const LayoutHeader = (props: Props) => {
+  const { classes, onShow, showSendFunds } = props
+  const address = useSelector(safeParamAddressFromStateSelector)
+  const granted = useSelector(grantedSelector)
+  const name = useSelector(safeNameSelector)
+  return !address ? null : (
```

Comment:
I would write
```
if (!address) return null
```

This is just my preference, I don't like how ternary looks there. Feel free to keep it if you like